datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/__tests__/filterSchemaRows.test.ts

```diff
@@ -35,6 +39,52 @@ describe('filterSchemaRows', () => {
         expect(expandedRowsFromFilter).toMatchObject(new Set());
     });
 
+    it('should properly filter schema rows based on description', () => {
+        const filterText = 'test';
+        const editableSchemaMetadata = { editableSchemaFieldInfo: [] };
+        const { filteredRows, expandedRowsFromFilter } = filterSchemaRows(
+            rows,
+            editableSchemaMetadata,
+            filterText,
+            testEntityRegistry,
+        );
+
+        expect(filteredRows).toMatchObject([{ fieldPath: 'testing' }]);
+        expect(expandedRowsFromFilter).toMatchObject(new Set());
+    });
+
+    it('should properly filter schema rows based on description regardless of capitalization', () => {
+        const editableSchemaMetadata = { editableSchemaFieldInfo: [] };
+        const filterText = 'TeSt';
+        const { filteredRows, expandedRowsFromFilter } = filterSchemaRows(
+            rows,
+            editableSchemaMetadata,
+            filterText,
+            testEntityRegistry,
+        );
+
+        expect(filteredRows).toMatchObject([{ fieldPath: 'testing' }]);
+        expect(expandedRowsFromFilter).toMatchObject(new Set());
+    });
+
+    it('should properly filter schema rows based on editable description', () => {
+        const editableSchemaMetadata = {
+            editableSchemaFieldInfo: [
+                { fieldPath: 'customer', description: 'customer description', globalTags: null, glossaryTerms: null },
+            ],
+        };
+        const filterText = sampleTag.properties.name;
```

Comment:
can you make this description in `editableSchemaFieldInfo` something different to make sure we're testing that it's matching specifically on that? so you can change it to `description: 'editable customer description'` and then make the filter text `editable customer description`

Reply:
then can you add one more test that's just like this but checking that we're filtering regardless of capitalization?